justfile

```
@@ -169,9 +169,11 @@ env-info:
     {{just_executable()}} --version
     rustc --version
     cargo --version
+    cargo --list
     rustup --version
     @echo "RUSTFLAGS='$RUSTFLAGS'"
     @echo "RUSTDOCFLAGS='$RUSTDOCFLAGS'"
+    @echo "PATH='$PATH'"
```

add an debug log for home

```suggestion
    @echo "PATH='$PATH'"
    @echo "HOME='$HOME'"
```